spring-reactive-modules/spring-5-reactive/src/main/java/com/baeldung/websocket/WebSocketController.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.websocket;
+
+import javax.websocket.*;
+import javax.websocket.server.ServerEndpoint;
+import java.io.IOException;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+@ServerEndpoint("/event-emitter")
+public class WebSocketController {
+
+    @OnOpen
+    public void onOpen(Session session, EndpointConfig endpointConfig) throws IOException {
+        // Get session and WebSocket connection
+        session.setMaxIdleTimeout(0);
+        System.out.println("Get session and WebSocket connection");
```

Comment:
why not to use some logger?

Reply:
I have updated to use logger.

Reply:
To have the file formatted similarly. Before each @Bean there is an empty line, so there should be also.